docs/src/faq.md

```diff
@@ -2,25 +2,36 @@
 
 ## Supported autodiff backends
 
-- Forward mode: [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl)
-- Reverse mode: all the packages compatible with [ChainRules.jl](https://github.com/JuliaDiff/ChainRules.jl)
+| Mode    | Backend                                                    | Support |
+| ------- | ---------------------------------------------------------- | ------- |
+| Forward | [ForwardDiff.jl]                                           | yes     |
+| Reverse | [ChainRules.jl]-compatible ([Zygote.jl], [ReverseDiff.jl]) | yes     |
```

Comment:
In another PR we should support ReverseDiff by default, right now it's opt-in. In an extension, we can make it inherit the rrule.

Reply:
I'll leave it to you, I think you have a much clearer idea of what to do than me

Reply:
https://github.com/gdalle/ImplicitDifferentiation.jl/issues/75